Experimental/DistributedAuthoritySample/Assets/Scripts/GameManagement/GameplayEventHandler.cs

```diff
@@ -0,0 +1,47 @@
+using Unity.Netcode;
+using UnityEngine;
+using System;
+using System.Threading.Tasks;
+
+namespace Unity.Multiplayer.Samples.SocialHub.GameManagement
+{
+    abstract class GameplayEventHandler
```

Comment:
this should probably be static

Reply:
Ah good catch. I've added this to our MathUtils class too.